backend/src/middleware/handleHtmlContent/handleContentData.js

```diff
@@ -0,0 +1,69 @@
+import extractMentionedUsers from './notifications/extractMentionedUsers'
+import extractHashtags from './hashtags/extractHashtags'
+
+const notify = async (postId, idsOfMentionedUsers, context) => {
```

Comment:
This part should be centralized in some notification handler - certainly not necessary to do this in this PR - but might be wise to open an Issue for that

Reply:
I do not understand, what exactly you mean with a notification handler. @ulfgebhardt 
When we come to mentions in Comments, we have to do something about that, may be.

Reply:
i am talking about the `notify` function. We should centralize everything which has to do with notifications. And as I stated - not necessary in this PR